config/betamocks/services_config.yml

```diff
@@ -894,6 +894,39 @@
         :uid_location: query
         :uid_locator: patient
 
+# EPS (Enhanced Patient Services) API
+- :name: "EPS API"
+  :base_uri: <%= "#{URI(Settings.vaos.eps.api_url).host}:#{URI(Settings.vaos.eps.api_url).port}" %>
+  :endpoints:
+    # Get a provider's service details
+    - :method: :get
```

Comment:
Note: These are not all the mocks we need, but I need other changes to move further into the workflow.